### PR TITLE
Update the menu for the User Input Elements

### PR DESCRIPTION
**Pull request by singa2000**
[Issue] N/A
[Problem] There was existing outdate examples
[Solution] Rearrange the sample list

Signed-off-by: cookie <cookie@samsung.com>

### CODE REVIEW

File: examples/wearable/UIComponents/contents/user-inputs-elements/button/bottom-button.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+<head>
+	<meta content="width=device-width, initial-scale=1, user-scalable=no" name="viewport" />
+	<title>
+		Wearable UI
+	</title>
+	<link href="../../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../lib/tau/wearable/theme/default/tau.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
+	<link href="../../../css/style.css" rel="stylesheet" />
+	<link href="../../../css/style.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
+	<script src="../../../js/circle-helper.js">
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/user-inputs-elements/button/bottom-button.html:13]: Is there an indentation in the code rule?
If there is no content, how about reducing the line?

File: examples/wearable/UIComponents/contents/user-inputs-elements/button/bottom-button.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+		</div>
+		<footer class="ui-footer ui-bottom-button ui-fixed">
+			<button class="ui-btn">
+				OK
+			</button>
+		</footer>
+	</div>
+</body>
+
+</html>
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/user-inputs-elements/button/bottom-button.html]: add to end-line.

File: examples/wearable/UIComponents/contents/user-inputs-elements/button/index.html [hunk 3]
```diff
@@ -26,40 +26,51 @@ <h2>
...
+						Long Text
 					</a>
 				</li>
+				<li>
+					<a href="short-text-button.html">
+						Short Text
+					</a>
+				</li>
+				<li>
+					<a href="selectmode.html">
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/user-inputs-elements/button/index.html]: How about changing to select-mode.html?

File: examples/wearable/UIComponents/contents/user-inputs-elements/button/selectmode.html [hunk 4]
```diff
@@ -6,11 +6,11 @@
...
-	<link href="../../css/style.css" rel="stylesheet" />
-	<link href="../../css/style.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
-	<script src="../../js/circle-helper.js">
+	<link href="../../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../lib/tau/wearable/theme/default/tau.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
+	<link href="../../../css/style.css" rel="stylesheet" />
+	<link href="../../../css/style.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
+	<script src="../../../js/circle-helper.js">
 	</script>
 	<script src="../../lib/tau/wearable/js/tau.js">
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/user-inputs-elements/button/selectmode.html]: Is the path correct?

File: examples/wearable/UIComponents/contents/user-inputs-elements/index.html [hunk 5]
```diff
@@ -22,11 +22,25 @@
...
+                <li>
+                    <a href="button/index.html">Button</a>
+                </li>
+                <li>
+                    <a href="controls/index.html">Controls</a>
+                </li>
+                <li>
+                    <a href="textinput/text-input.html">Text inputs</a>
+                </li>
+           		<li>
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/user-inputs-elements/index.html]: check to indent.

File: examples/wearable/UIComponents/contents/user-inputs-elements/pickers/date-picker.html [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+		<div class="ui-content">
+			<div class="ui-date-picker">
+			</div>
+		</div>
+		<script src="date-picker.js">
+		</script>
+	</div>
+</body>
+
+</html>
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/user-inputs-elements/pickers/date-picker.html]: add end-line.

File: examples/wearable/UIComponents/contents/user-inputs-elements/pickers/index.html [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+	<link href="../../../css/style.css" rel="stylesheet" />
+	<link href="../../../css/style.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
+	<script src="../../../js/circle-helper.js">
+	</script>
+	<script src="../../../lib/tau/wearable/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page" data-go-to-top-button="true">
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/user-inputs-elements/pickers/index.html:20]: just question.

What does 'data-go-to-top-button' mean?
How does it change when false?

File: examples/wearable/UIComponents/contents/user-inputs-elements/pickers/time-picker.html [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+		<div class="ui-content">
+			<div class="ui-time-picker" data-format="h">
+			</div>
+		</div>
+		<script src="time-picker.js">
+		</script>
+	</div>
+</body>
+
+</html>
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/user-inputs-elements/pickers/time-picker.html]: add end-line.

File: examples/wearable/UIComponents/contents/user-inputs-elements/pickers/time-picker24.html [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+		<div class="ui-content">
+			<div class="ui-time-picker" data-format="H">
+			</div>
+		</div>
+		<script src="time-picker24.js">
+		</script>
+	</div>
+</body>
+
+</html>
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/user-inputs-elements/pickers/time-picker24.html]: ditto.

File: examples/wearable/UIComponents/contents/user-inputs-elements/textinput/text-input.html [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+		<script src="text-input.js">
+		</script>
+	</div>
+	<script src="../../../js/app.js">
+	</script>
+	<script src="../../../js/lowBatteryCheck.js">
+	</script>
+</body>
+
+</html>
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/user-inputs-elements/textinput/text-input.html]: ditto.

**Review by DohyungLim — COMMENTED** [review]
10 comment.

File: examples/wearable/UIComponents/contents/user-inputs-elements/pickers/index.html (hunk 7 above)
Comment (singa2000) [reply examples/wearable/UIComponents/contents/user-inputs-elements/pickers/index.html:20]: It's button to go to the top of the page.
It's new feature.
If you set the value it by "false", the button is not display.

File: examples/wearable/UIComponents/contents/user-inputs-elements/pickers/time-picker.html (hunk 8 above)
Comment (singa2000) [reply examples/wearable/UIComponents/contents/user-inputs-elements/pickers/time-picker.html]: Done

File: examples/wearable/UIComponents/contents/user-inputs-elements/pickers/time-picker24.html (hunk 9 above)
Comment (singa2000) [reply examples/wearable/UIComponents/contents/user-inputs-elements/pickers/time-picker24.html]: Done

File: examples/wearable/UIComponents/contents/user-inputs-elements/textinput/text-input.html (hunk 10 above)
Comment (singa2000) [reply examples/wearable/UIComponents/contents/user-inputs-elements/textinput/text-input.html]: Done

File: examples/wearable/UIComponents/contents/user-inputs-elements/button/index.html (hunk 3 above)
Comment (singa2000) [reply examples/wearable/UIComponents/contents/user-inputs-elements/button/index.html]: sure. I changed

File: examples/wearable/UIComponents/contents/user-inputs-elements/button/selectmode.html (hunk 4 above)
Comment (singa2000) [reply examples/wearable/UIComponents/contents/user-inputs-elements/button/selectmode.html]: Thanks. I changed.

File: examples/wearable/UIComponents/contents/user-inputs-elements/button/bottom-button.html (hunk 1 above)
Comment (singa2000) [reply examples/wearable/UIComponents/contents/user-inputs-elements/button/bottom-button.html:13]: I'm not sure about the correct indentation code rule.
However, in our project, we're using the <script> tag with linefeed.
So I think it's better keep going to current status.

File: examples/wearable/UIComponents/contents/user-inputs-elements/button/index.html [hunk 11]
```diff
@@ -26,40 +26,51 @@ <h2>
...
-						Short Text
+					<a href="small-icon-button.html">
+						Small Icon
 					</a>
 				</li>
 				<li>
-					<a href="long-text-button.html">
-						Long Text
+					<a href="icon-button.html">
+						Icon
```
Comment (HunseopJeong) [inline examples/wearable/UIComponents/contents/user-inputs-elements/button/index.html]: How about renaming the 'Icon Button'?

File: examples/wearable/UIComponents/contents/user-inputs-elements/button/index.html [hunk 12]
```diff
@@ -26,40 +26,51 @@ <h2>
...
-						Icon
+					<a href="icon-button-from-js.html">
+						Icon From JS
 					</a>
 				</li>
 				<li>
-					<a href="small-icon-button.html">
-						Small Icon
+					<a href="bottom-button.html">
+						Bottom button
```
Comment (HunseopJeong) [inline examples/wearable/UIComponents/contents/user-inputs-elements/button/index.html:50]: Is it good to sync the naming rule?
Appendix all 'button' or remove the 'button'.

**Review by HunseopJeong — APPROVED** [review]
LGTM

File: examples/wearable/UIComponents/contents/user-inputs-elements/button/index.html (hunk 12 above)
Comment (singa2000) [reply examples/wearable/UIComponents/contents/user-inputs-elements/button/index.html:50]: I appendix all "button" string.
Thanks.

**Review by DohyungLim — APPROVED** [review]
LGTM.

**Pull request merged**